app/api/todos/route.ts

```diff
@@ -0,0 +1,16 @@
+"use server";
+import { NextResponse } from "next/server";
+import connectDB from "../../../lib/mongodb";
+import Todo from "../models/todosModel";
+
+export async function GET() {
+  await connectDB();
+
+  try {
+    const todos = await Todo.find({});
+    return NextResponse.json({ data: todos, success: "true" });
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
we don't disable eslint :)
If you don't want to use error in catch, you can add underscore before it like `_error`